main.nf

```diff
@@ -1,79 +1,83 @@
 #!/usr/bin/env nextflow
-
 /*
 ========================================================================================
-                  C H I P - S E Q   B E S T   P R A C T I C E
+                         nf-core/chipseq
 ========================================================================================
- ChIP-seq Best Practice Analysis Pipeline. Started May 2016.
+ nf-core/chipseq Analysis Pipeline.
  #### Homepage / Documentation
  https://github.com/nf-core/chipseq
- @#### Authors
- Chuan Wang <chuan.wang@scilifelab.se>
- Phil Ewels <phil.ewels@scilifelab.se>
- Alex Peltzer <alexander.peltzer@qbic.uni-tuebingen.de>
- Harshil Patel <harshil.patel@crick.ac.uk>
 ----------------------------------------------------------------------------------------
 */
 
 def helpMessage() {
     log.info nfcoreHeader()
-    log.info """
-
+    log.info"""
     Usage:
 
     The typical command for running the pipeline is as follows:
 
-      nextflow run nf-core/chipseq --reads '*_R{1,2}.fastq.gz' --genome GRCh37 --macsconfig 'macssetup.config' -profile docker
+      nextflow run nf-core/chipseq --design design.csv --genome GRCh37 -profile docker
 
     Mandatory arguments:
-      --reads                       Path to input data (must be surrounded with quotes)
+      --design                      Comma-separted file containing information about the samples in the experiment (see docs/usage.md)
```

Comment:
```suggestion
      --design                      Comma-separated file containing information about the samples in the experiment (see docs/usage.md)
```

Reply:
Fixed locally! Will come in with the next commit. Thanks. Didnt spot that :+1: